src/psyclone/psyGen.py

```diff
@@ -2497,13 +2497,17 @@ def zero_reduction_variable(self, parent, position=None):
         local_var_name = self.local_reduction_name
         var_type = self._reduction_arg.type
         if var_type == "gh_real":
-            zero = "0.0_r_def"
-            kind_type = "r_def"
             data_type = "real"
+            data_value = "0.0"
```

Comment:
Not your fault but pycodestyle complains about a missing blank line at line 74 of this file. Please could you add one?

Reply:
Added.